Tests/scripts/hook_validations/secrets.py

```diff
@@ -105,23 +94,15 @@ def get_diff_text_files(files_string):
         file_data = file_name.split()
         if not file_data:
             continue
-
         file_status = file_data[0]
-        if file_status.upper() == "R099":
-            # if filename renamed
-            # sometimes the R comes with numbers R099,
-            # the R status file usually will look like:
-            # R099 TestsPlaybooks/foo.yml TestPlaybooks/playbook-foo.yml
-            # that is why we set index 2 to file_path - the second index is the updated file name
+        if 'r' in file_status.lower():
             file_path = file_data[2]
         else:
             file_path = file_data[1]
-
         # only modified/added file, text readable, exclude white_list file
-        if file_status.upper() in ACCEPTED_FILE_STATUSES and is_text_file(file_path):
+        if (file_status.lower() in ACCEPTED_FILE_STATUSES or 'r' in file_status.lower()) and is_text_file(file_path):
```

Comment:
`ACCEPTED_FILE_STATUSES` is uppercase...

Reply:
lol so I didn't change it in the end :s